tests/bson/bson-symbol-compare-001.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+MongoDB\BSON\Symbol comparisons
+--FILE--
+<?php
+
+var_dump(MongoDB\BSON\toPHP(MongoDB\BSON\fromJSON('{ "undefined": {"$symbol": "val1"} }')) == MongoDB\BSON\toPHP(MongoDB\BSON\fromJSON('{ "undefined": {"$symbol": "val1"} }')));
```

Comment:
"undefined" is a curious choice here. Something like "x" or "foo" might be more concise, and not cause any confusion with the actual Undefined type.

Edit: I noticed you used "symbol" in a later test. That certainly works.